examples/custom_vocabularies_example.py

```diff
@@ -0,0 +1,36 @@
+"""Copyright 2019 REV
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from rev_ai import customvocabulariesclient
+
+access_token = "your_access_token"
+
+print("Creating Custom Vocab Client")
```

Comment:
I think you can reduce the number of new lines in this file. Newlines only really need to go between "logical blocks" meaning sections of code with different purposes